src/pages/layer-2.js

```diff
@@ -0,0 +1,1099 @@
+        <p>
+          Generalized layer 2s behave just like Ethereum — but cheaper. Anything
+          that you can do on Ethereum layer 1, you can also do on layer 2. Many
+          dapps have already begun to migrate to these networks, or are skipping
+          deploying to Mainnet altogether.
+        </p>
+        <CardGrid>
+          {layer2DataCombined
+            .filter((l2) => !l2.purpose.indexOf("universal"))
+            .map((l2, idx) => {
+              return (
+                <Layer2ProductCard
+                  key={idx}
+                  background={l2.background}
+                  image={getImage(data[l2.imageKey])}
+                  description={l2.description}
+                  url={l2.website}
+                  note={translateMessageId(l2.noteKey, intl)}
+                  name={l2.name}
+                  bridge={l2.bridge}
+                  ecosystemPortal={l2.ecosystemPortal}
+                  tokenLists={l2.tokenLists}
+                />
+              )
+            })}
+        </CardGrid>
+      </PaddedContent>
+
+      <PaddedContent>
+        <h2>Application specific layer 2s</h2>
+        <p>
+          Application specific layer 2s are projects that specialize in
+          optimizing for a specific application space, bringing improved
+          performance.
+        </p>
+        <CardGrid>
+          {layer2DataCombined
+            .filter((l2) => l2.purpose.indexOf("universal"))
+            .map((l2, idx) => {
+              return (
+                <Layer2ProductCard
+                  key={idx}
+                  background={l2.background}
+                  image={getImage(data[l2.imageKey])}
+                  description={l2.description}
+                  url={l2.website}
+                  note={translateMessageId(l2.noteKey, intl)}
+                  name={l2.name}
+                  bridge={l2.bridge}
+                  ecosystemPortal={l2.ecosystemPortal}
+                  tokenLists={l2.tokenLists}
+                >
+                  {l2.purpose.map((purpose) => (
+                    <Pill>{purpose}</Pill>
+                  ))}
+                </Layer2ProductCard>
+              )
+            })}
+        </CardGrid>
+      </PaddedContent>
+
+      <PaddedContent>
+        <h2>A note on alt L1s, sidechains, and validiums</h2>
+        <TwoColumnContent>
+          <Flex50>
+            <p>
+              <b>Alternative layer 1s</b> have higher throughput and lower
+              transaction fees than Ethereum. These alt L1s have had to{" "}
+              <b>sacrifice on security or decentralization</b> in order to
+              achieve higher transactions per second and lower fees. The
+              Ethereum ecosystem is firmly aligned that{" "}
+              <b>
+                layer 2 scaling is the only way to solve the scalability
+                trilemma
+              </b>{" "}
+              and remain decentralized and secure.
```

Comment:
Small though, felt this read better:
```suggestion
              while remaining decentralized and secure.
```